photon-lib/src/main/scala/com/linkedin/photon/ml/hyperparameter/estimators/kernels/StationaryKernel.scala

```diff
@@ -14,25 +14,39 @@
  */
 package com.linkedin.photon.ml.hyperparameter.estimators.kernels
 
-import breeze.linalg.{DenseMatrix, DenseVector, *, squaredDistance}
-import breeze.numerics.log
+import breeze.linalg._
+import breeze.numerics.constants.Pi
+import breeze.numerics.{log, pow, sqrt}
+
+import com.linkedin.photon.ml.hyperparameter.Linalg.choleskySolve
 
 /**
  * Base trait for stationary covariance kernel functions
  *
  * Stationary kernels depend on the relative positions of points (e.g. distance), rather than on their absolute
  * positions.
  *
+ * @param amplitude the covariance amplitude
+ * @param noise the observation noise
  * @param lengthScale the length scale of the kernel. This controls the complexity of the kernel, or the degree to which
  *   it can vary within a given region of the function's domain. Higher values allow less variation, and lower values
  *   allow more.
- * @param lengthScaleBounds the bounds within which the length scale must fall
  */
 abstract class StationaryKernel(
-    lengthScale: DenseVector[Double] = DenseVector(1.0),
-    lengthScaleBounds: (Double, Double) = (1e-5, 1e5))
+    amplitude: Double = 1.0,
+    noise: Double = 1e-3,
+    lengthScale: DenseVector[Double] = DenseVector(1.0))
   extends Kernel {
 
+  // Amplitude lognormal prior
+  val amplitudeScale = 1.0
```

Comment:
Should these be updated as the iterations grow? Keeping them fixed might issues.

Reply:
I defaulted to spearmint's implementation, where the priors are also fixed like this. I agree that it's not ideal, but probably a reasonable starting point? We still get a posterior based on observations as iterations grow.